packages/sitecore-jss-dev-tools/src/clean.test.ts

```diff
@@ -31,7 +31,7 @@ describe('clean', () => {
         },
       },
     ];
-    const delStub = sinon.stub(del, 'sync').returns(mockPaths);
+    const delStub = sinon.stub(del, 'sync').returns((mockPaths as any) as string[]);
```

Comment:
Implementation for this unit test is incorrect in terms of expected values, can you, please apply these changes, instead of applying a workaround for the type issue?
```ts
  it('should clean', () => {
    const path = 'c:/test';
    const mockPaths = [`${path}/index.ts`];
    const delStub = sinon.stub(del, 'sync').returns(mockPaths);

    const logSpy = sinon.spy(console, 'log');
    clean({ path: path });

    expect(delStub.called).to.be.true;
    expect(logSpy.calledWith('Cleaned:\n', mockPaths.join('\n'))).to.be.true;
  });
```

Reply:
Sure, sounds good.